src/test/lib/DataTest.cpp

```diff
@@ -3355,9 +3355,9 @@ QuicTestStreamReliableReset(
     TEST_QUIC_SUCCEEDED(ClientConfiguration.GetInitStatus());
 
     StreamReliableReset Context;
-    uint8_t SendDataBuffer[BUFFER_SIZE];
+    uint8_t *SendDataBuffer = new(std::nothrow) uint8_t[BUFFER_SIZE];
```

Comment:
Need to make sure this gets freed at the end. I think we have a `UniquePtr` helper already.

Reply:
Need a `UniquePtr` here too (just like below).

Reply:
silly me